### DISCUSSION TITLE
Symfony \ Component \ Debug \ Exception \ FatalThrowableError (E_ERROR) Call to undefined function Illuminate\Database\Connectors\PDO()

### DISCUSSION THREAD

**Discussion by Irewoleo** [laravel/framework#32985] [General]
I get this error when trying to sign in on laravel built platform and I got this as an error.

Symfony \ Component \ Debug \ Exception \ FatalThrowableError (E_ERROR)
Call to undefined function Illuminate\Database\Connectors\PDO()

use Illuminate\Support\Facades\Auth;

class RedirectIfAuthenticated
{
/**
* Handle an incoming request.
*
* @param \Illuminate\Http\Request $request
* @param \Closure $next
* @param string|null $guard
* @return mixed
*/
public function handle($request, Closure $next, $guard = null)
{
if (Auth::guard($guard)->check()) {
return redirect('/home');
}

    return $next($request);
}
}

Arguments
Request {#43 ▼
#json: null
#convertedFiles: []
#userResolver: Closure($guard = null) {#260 …5}
#routeResolver: Closure() {#262 …5}
+attributes: ParameterBag {#45}
+request: ParameterBag {#44}
+query: ParameterBag {#51}
+server: ServerBag {#47}
+files: FileBag {#48}
+cookies: ParameterBag {#46}
+headers: HeaderBag {#49}
#content: null
#languages: null
#charsets: null
#encodings: null
#acceptableContentTypes: array:6 [▶]
#pathInfo: "/login"
#requestUri: "/public_html/login"
#baseUrl: "/public_html"
#basePath: null
#method: "POST"
#format: null
#session: Store {#289}
#locale: null
#defaultLocale: "en"
-isHostValid: true
-isForwardedValid: true
basePath: "/public_html"
format: "html"
}

Can someone help me out?

**Comment by GrahamCampbell** [comment]
Not without a stack trace. Though, at a guess it looks like you don't have `ext-pdo` installed.
